projects/addon-editor/components/toolbar-tools/list-configs/list-configs.component.ts

```diff
@@ -0,0 +1,39 @@
+import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
+import {TuiEditor} from '@taiga-ui/addon-editor/abstract';
+import {TuiTiptapEditorService} from '@taiga-ui/addon-editor/directives';
+import {TUI_EDITOR_TOOLBAR_TEXTS} from '@taiga-ui/addon-editor/tokens';
+import {LanguageEditor} from '@taiga-ui/i18n';
+import {combineLatest, Observable} from 'rxjs';
+import {distinctUntilChanged, map, startWith} from 'rxjs/operators';
+
+@Component({
+    selector: 'tui-list-configs',
+    templateUrl: './list-configs.template.html',
+    styleUrls: ['../tools-common.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TuiListConfigsComponent {
+    readonly listState$ = combineLatest([
+        this.getActiveStatus$('orderedList'),
+        this.getActiveStatus$('bulletList'),
+    ]).pipe(
+        map(([ordered, unordered]) => ({
+            ordered,
+            unordered,
+        })),
+        startWith({ordered: false, unordered: false}),
```

Comment:
Why only this case has a `startWith`? Is it necessary?

Reply:
it is required here because it could be asked before the first observable emit (and we get error with attempt to take property from `null`).

https://github.com/TinkoffCreditSystems/taiga-ui/blob/52990bf63b2d4be7d94f7b54491b39cce4bddf86/projects/addon-editor/components/toolbar-tools/list-configs/list-configs.template.html#L19